pyccel/ast/core.py

```diff
@@ -1346,8 +1346,16 @@ def __init__(
         self._iterable = iter_obj
         self._body = body
         self._local_vars = local_vars
+        self._nowait_expr = None
         super().__init__()
 
+    @property
+    def nowait_expr(self):
+        return self._nowait_expr
+
+    def set_nowait_expr(self, expr):
+        self._nowait_expr = expr
+
```

Comment:
can't you use setter decorator here ?
```Python
@nowait_expr.setter
```

Reply:
when i use the setter decorator i get this error : `TypeError: 'NoneType' object is not callable`

Reply:
fixed